tests/e2e/pageObjects/browser-page.ts

```diff
@@ -1013,6 +1014,39 @@ export class BrowserPage {
             .click(this.selectIndexDdn)
             .click(option);
     }
+
+    /**
+    * Get text from first tree element
+    */
+
+    async getTextFromFirstTreeElement(): Promise<String> {
+        return (await Selector(`[role="treeitem"]`).nth(0).find(`div`).textContent).replace(/\s/g, '');
+    }
+
+    /**
+    * Get text from first tree element
+    * @param names folder names with sequence of subfolder
+    * Example: if names ['mobile', '2']
+    * It will go to mobile -> 2 -> keys
```

Comment:
IMO it's not required. We generally do that where we have "complexity."
@vlad-dargel - WDYT?